components/Header/index.js

```diff
@@ -12,7 +12,7 @@ import { classNames, navigationItems } from '/components/utils';
 
 export default function Header(props) {
   return (
-    <Popover className={` relative border-b-2 border-gray-50 ${props.className}`}>
+    <Popover className={` relative border-b-2 border-gray-200 bg-gray-100 dark:border-slate-600 dark:bg-slate-800 ${props.className}`}>
```

Comment:
Personally, I'm not a huge fan of the slightly darker gray background for the menu bar. We could do a quick poll in the `website-admin` channel to see. If most people like it, we can run with it.

Reply:
yeah, lets do that

Reply:
Just posted to the channel to get some additional thoughts!